packages/phoenix-evals/src/phoenix/evals/classify.py

```diff
@@ -268,6 +270,126 @@ def _run_llm_classification_sync(input_data: pd.Series[Any]) -> ParsedLLMRespons
     )
 
 
+def audio_classify(
+    dataframe: Union[List, pd.DataFrame],
+    model: BaseModel,
+    template: Union[ClassificationTemplate, PromptTemplate, str],
+    rails: List[str],
+    data_fetcher: Optional[Callable[[str], Audio]],
```

Comment:
I still don't think we should be requiring the user return our internal type, it feels like a structure they need to learn / import from that feels clunky